updatePipelines.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+IMAGE_BASENAME=gps-to-ads-app
```

Comment:
Could you please put all defines to the top and give them good names?

Reply:
Improved updatePipelines.sh by using a loop. This way we can easily add more demos.
Unfortunately, rebasing fails for some reason...